### PR TITLE
Support for New Versions of Django

### PR DESCRIPTION
**Pull request by pvanfas**

### CODE REVIEW

File: freeze/__init__.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
 # -*- coding: utf-8 -*-
 
 from freeze.version import __version__
-
```
Comment (fabiocaccamo) [inline freeze/__init__.py:4]: Why?

File: freeze/version.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
 # -*- coding: utf-8 -*-
 
-__version__ = '0.6.4'
-
+__version__ = "0.7"
```
Comment (fabiocaccamo) [inline freeze/version.py:3]: Don't worry about the library version.

File: freeze/urls.py [hunk 3]
```diff
@@ -1,12 +1,17 @@
 # -*- coding: utf-8 -*-
 
-from django.conf.urls import url
-
+from django.urls import path
```
Comment (fabiocaccamo) [inline freeze/urls.py:3]: Please add conditional import depending on django version for bacward-compatibility.

**Review by fabiocaccamo — CHANGES_REQUESTED** [review]

**Comment by fabiocaccamo** [comment]
Why you closed this?

**Comment by pvanfas** [comment]
i will reopen with backward compatibility changes

**Pull request closed without merging**